components/permalink/share-url.service.ts

```diff
@@ -94,7 +97,7 @@ export class HsShareUrlService {
     this.HsUtilsService.debounce(
       () => {
         this.Location.replaceState(
-          this.Location.path(),
```

Comment:
Seems I was too quick to approve this. Now we have a problem that if the page is located in a subdirectory, hslayers rewrites the url by removing the directory, for example https://local.polirural-mb.lv/private/?hs_x=28... becomes https://local.polirural-mb.lv/?hs_x=2865324.91..

Reply:
Fixed this in https://github.com/hslayers/hslayers-ng/pull/1012/commits/cd17e1ae933dc81dc20f6c9dd7a5b796a5908194

Reply:
Oh, I didn't realised that this can happen. I was too quick in my assumptions too.